src/slotmap.js

```diff
@@ -316,10 +316,11 @@ class PersistentStorageMap {
      * @param {number} blockNumber
      * @param {bigint} slot
      * @param {Value} value
+     * @param {string} type
```

Comment:
for now let's use a more restrictive type

```suggestion
     * @param {'t_string_storage' | 't_bytes_storage'} type
```